.gitignore

```diff
@@ -7,7 +7,6 @@ __pycache__/
 # Django stuff & migrations
 local_settings.py
 db.sqlite3
-**/migrations/
```

Comment:
Cette ligne empêche d'avoir les migrations de django dans GIT (tu dois les avoir normalement)

Reply:
J'ai eu cette erreur du coup :

`(PYPO) $(ARIANE)> python manage.py makemigrations
Migrations for 'artists':
  artists/migrations/0001_initial.py
    - Create model ArtistPage
    - Create model ArtistsCatalogPage
    - Create model ArtistPageYoutubeLink
    - Create model ArtistPageWebsiteLink
    - Create model ArtistPageTwitterLink
    - Create model ArtistPagePlayer
    - Create model ArtistPageInstagramLink
    - Create model ArtistPageGigs
    - Create model ArtistPageFacebookLink
Migrations for 'blog':
  blog/migrations/0001_initial.py
    - Create model BlogIndexPage
    - Create model BlogPage
    - Create model BlogTagIndexPage
    - Create model BlogPageTag
    - Create model BlogPageGalleryImage
    - Add field tags to blogpage
Migrations for 'events':
  events/migrations/0001_initial.py
    - Create model EventsIndexPage
    - Create model EventPage
Migrations for 'home':
  home/migrations/0003_auto_20200824_1705.py
    - Create model TeamPage
    - Create model TermsPage
    - Change Meta options on homepage
    - Add field agenda_text to homepage
    - Add field banner to homepage
    - Add field playlist_text to homepage
    - Create model TeamPageMembers
    - Create model HomePagePlayer
    - Create model AssociationPage
(PYPO) $(ARIANE)> python manage.py migrate
Operations to perform:
  Apply all migrations: admin, artists, auth, blog, contenttypes, events, home, sessions, taggit, wagtailadmin, wagtailcore, wagtaildocs, wagtailembeds, wagtailforms, wagtailimages, wagtailredirects, wagtailsearch, wagtailusers
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/ariane/Dev/virtualenvs/PYPO/lib/python3.6/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/home/ariane/Dev/virtualenvs/PYPO/lib/python3.6/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/ariane/Dev/virtualenvs/PYPO/lib/python3.6/site-packages/django/core/management/base.py", line 328, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/ariane/Dev/virtualenvs/PYPO/lib/python3.6/site-packages/django/core/management/base.py", line 369, in execute
    output = self.handle(*args, **options)
  File "/home/ariane/Dev/virtualenvs/PYPO/lib/python3.6/site-packages/django/core/management/base.py", line 83, in wrapped
    res = handle_func(*args, **kwargs)
  File "/home/ariane/Dev/virtualenvs/PYPO/lib/python3.6/site-packages/django/core/management/commands/migrate.py", line 190, in handle
    pre_migrate_apps = pre_migrate_state.apps
  File "/home/ariane/Dev/virtualenvs/PYPO/lib/python3.6/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/ariane/Dev/virtualenvs/PYPO/lib/python3.6/site-packages/django/db/migrations/state.py", line 209, in apps
    return StateApps(self.real_apps, self.models)
  File "/home/ariane/Dev/virtualenvs/PYPO/lib/python3.6/site-packages/django/db/migrations/state.py", line 272, in __init__
    self.render_multiple([*models.values(), *self.real_models])
  File "/home/ariane/Dev/virtualenvs/PYPO/lib/python3.6/site-packages/django/db/migrations/state.py", line 315, in render_multiple
    "for more" % (new_unrendered_models, get_docs_version())
django.db.migrations.exceptions.InvalidBasesError: Cannot resolve bases for [<ModelState: 'contact.ContactPage'>]
This can happen if you are inheriting models from an app with migrations (e.g. contrib.auth)
 in an app with no migrations; see https://docs.djangoproject.com/en/3.0/topics/migrations/#dependencies for more`